manifests/master.yaml

```diff
@@ -0,0 +1,67 @@
+apiVersion: v1
```

Comment:
Are these (master.yaml, node.yaml) used in any of our current guides?

If not, we shouldn't merge them yet.